Quadriga/src/main/java/edu/asu/spring/quadriga/service/impl/publicwebsite/AboutTextManager.java

```diff
@@ -0,0 +1,47 @@
+package edu.asu.spring.quadriga.service.impl.publicwebsite;
+
+import org.hibernate.HibernateException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import edu.asu.spring.quadriga.dao.publicwebsite.IAboutTextDAO;
+import edu.asu.spring.quadriga.domain.settings.IAboutText;
+import edu.asu.spring.quadriga.dto.AboutTextDTO;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.service.publicwebsite.IAboutTextManager;
+
+/**
+ * Service to save title and description of public website about page form.
+ * 
+ * @author Rajat Aggarwal
+ *
+ */
+@Service
+public class AboutTextManager implements IAboutTextManager {
+
+	@Autowired
+	private IAboutTextDAO aboutTextDAO;
+
+	@Transactional
+	@Override
+	public void saveAbout(String projectId, String title, String description) throws QuadrigaStorageException {
+		AboutTextDTO aboutTextDTO;
+		aboutTextDTO = aboutTextDAO.getDTOByProjectId(projectId);
+		
+		if (aboutTextDTO == null)
+			aboutTextDTO = new AboutTextDTO();
+
+		aboutTextDTO.setProjectId(projectId);
+		aboutTextDTO.setTitle(title);
+		aboutTextDTO.setDescription(description);
+		aboutTextDAO.saveNewDTO(aboutTextDTO);
```

Comment:
This method calls "save" to store the object. Save is generally only for new objects and i assume it'll give you an exception when you try to save an object that was already saved. You should use the update method for about texts that were already stored.